docker-compose.yml

```diff
@@ -22,6 +22,31 @@ services:
       - "8080:8080"
       - "8081:8081"
 
+  db:
+    image: postgres:latest
+    networks:
+      - kafka_network
+    environment:
+      POSTGRES_DB: mydb
+      POSTGRES_USER: username
+      POSTGRES_PASSWORD: password
+    volumes:
+      - ./db/data:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
+
+  pgadmin:
+    image: dpage/pgadmin4:latest
+    networks:
+      - kafka_network
+    environment:
+      PGADMIN_DEFAULT_EMAIL: admin@example.com
+      PGADMIN_DEFAULT_PASSWORD: admin_password
+    ports:
+      - "5050:80"
+    depends_on:
+      - db
+
 networks:
   kafka_network:
     external: true
```

Comment:
You should create a new internal network (for example `zeevision_network`) which is used by Postgres and pgAdmin instead of `kafka_network`. You then also need to add `zeevision` itself to that network so Postgres is available to it.

This is more of a security thing because we don't want anyone else being able to connect to the database. `kafka_network` is used exclusively for connecting to Kafka.